en/assembly/cable_wiring.md

```diff
@@ -0,0 +1,186 @@
+
+'Pixhawk colors' are based on  [dronecode standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf) directly.
+
+Some connection types with the same connector require different wire pairing (for example, I2C and CAN signals).
+:::
+
+| Color  | Name  |  Preferred usage |
+|--------|------------------| --- |
+| ![red](../../assets/hardware/cables/red.png)   | Red  | Power voltage |     
+| ![green](../../assets/hardware/cables/green.png)    | Green   | General purpose signal |
+| ![white](../../assets/hardware/cables/white.png) | White |  General purpose signal |
+| ![yellow](../../assets/hardware/cables/yellow.png)   | Yellow            | General purpose signal |
+|  ![blue](../../assets/hardware/cables/blue.png)  | Blue            |  Power return, Open-collector control signal |
+| ![black](../../assets/hardware/cables/black.png)   | Black             |  GND, Power return ground |
+
+The assignment of signal wire colors in cables follows the rules:
+
+- Color of the signal does not repeat in the connector for wires adjacent to each other. (The color combination is always unique for signal cable type)
+- The same signal type has to the same color primarily
+- Wiring harnesses of the same pin count have to unique color sequence, which determines the cable type. (This is especially useful on photographs used in a manual)
+- The red and black colors are reserved for power
+
+:::Note
+
+The pin labels shown in follow description correspond to the pinout on the autopilot side. All cables are straight (1:1). If they require crossover (e.g. UART), this should be solved by internal connection of the devices.
+
+:::
+
+### I²C cables
+
+[I2C bus](https://en.wikipedia.org/wiki/I%C2%B2C) is widely used for sensor connection. The currently used cable colors are specified in following table.
+
+|Signal | Pixhawk Colors | ThunderFly colors | CUAV colors (I2C/CAN) |
+|--------|------------------|---------------------|-----------------------|
+| +5V  | Red             |   ![red](../../assets/hardware/cables/red.png) Red                   | Red                   |
+| SCL  | Black          |  ![yellow](../../assets/hardware/cables/yellow.png) Yellow            | White                |
+| SDA  | Black          |  ![green](../../assets/hardware/cables/green.png) Green               | Yellow               |
+| GND | Black          | ![black](../../assets/hardware/cables/black.png) Black                 | Black                 |
+
+The [Dronecode standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf) assumes a 1.5k ohm pull-up resistor on SDA and SCL signals in autopilot.
+
+#### Cable twisting

```

Comment:
Does the benefit of cable twisting only apply to I2C? As this only covers I2C perhaps put that into the heading.

Reply:
Ignore, sorry, didn't read ahead to see you cover this in each section